.gitlab/install_ddtrace_deps.rb

```diff
@@ -0,0 +1,78 @@
+# require "fileutils"
+require "open3"
+require "bundler"
+
+lock_file_path = "./Gemfile.lock"
+install_dir = "./vendor"
+
+# lock_file_path = "/opt/datadog/apm/library/ruby/Gemfile.lock"
+# install_dir = "./vendor"
```

Comment:
Is this commented out code helpful?
If so, we should document when to uncomment it.